apps/studio/components/interfaces/App/CommandMenu/Support.tsx

```diff
@@ -0,0 +1,41 @@
+import { LifeBuoy } from 'lucide-react'
+import { useMemo } from 'react'
+
+import { IS_PLATFORM } from 'common'
+import type { ICommand } from 'ui-patterns/CommandMenu'
+import { useRegisterCommands } from 'ui-patterns/CommandMenu'
+import { COMMAND_MENU_SECTIONS } from './CommandMenu.utils'
+
+const useSupportCommands = () => {
+  const commands = useMemo(
+    () =>
+      [
+        {
+          id: 'support',
+          name: 'Support',
+          route: '/support',
```

Comment:
this gives me a 404 — prob just on staging? 
https://studio-staging-git-feat-new-command-menu-studio-supabase.vercel.app/dashboard/support

Reply:
Ahhh, no, good catch, this is under www not studio so needs to be a full route